examples/tutorials/Differentiation_Infrastructure_in_Deepchem.ipynb

```diff
@@ -0,0 +1,417 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# **Differentiation Infrastructure in Deepchem**\n",
+    "\n",
+    "Author : Rakshit Kr. Singh : [Website](https://greatrsingh.in/) : [LinkedIn](https://www.linkedin.com/in/rakshit-singh-ai/) : [GitHub](https://github.com/GreatRSingh)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Differentiation Methods like Broyden's Methods is capable of allowing solutions in non-archimedean settings that's why it is widely used in the field of Quantum Chemistry. Additionally, Broyden's method has been successfully applied to accelerate the convergence of nonlinear equations systems in semiconductor device simulations, outperforming other methods in terms of accuracy and iteration steps. **Deepchem** is moving towards leveraging these capabilities to solve complex problems in the field of Quantum Chemistry and Drug Discovery."
```

Comment:
@GreatRSingh minor grammatical corrections here: 
- Differentiation methods, like Broyden's methods, are capable of allowing solutions in non-Archimedean settings, which is why it is widely used in the field of quantum chemistry......
- "DeepChem" instead "Deepchem"
- ....quantum chemistry and drug discovery. (no need to capitatlize these)